### PR TITLE
Added remove meshes stage

### PR DESCRIPTION
**Pull request by leerichard42**

### CODE REVIEW

File: specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf [hunk 1]
```diff
@@ -159,6 +159,14 @@
                     "mode": 4
                 }
             ]
+        },
+        "unusedMeshId": {
+            "name": "Mesh",
```
Comment (pjcozzi) [inline specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf]: `name` is optional so I would remove here and in any other new examples.

File: specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf [hunk 2]
```diff
@@ -159,6 +159,14 @@
...
+            "primitives": [
```
Comment (pjcozzi) [inline specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf]: `primitives` is also optional so we could just make this an empty object.

See https://github.com/KhronosGroup/glTF/blob/master/specification/README.md#reference-mesh

**Comment by pjcozzi** [comment]
Just those minor comments.

**Comment by leerichard42** [comment]
Updated!

**Comment by pjcozzi** [comment]
Thanks!

**Pull request merged**